docs/user/metacat/source/intro.rst

```diff
@@ -69,22 +66,16 @@ What's in this Guide
 --------------------
 This Administrator's guide includes information for installing, configuring,
 managing and extending Metacat for Ubuntu and other Linux variants. Metacat
-can run on MacOS but it needs users' customized installation. 
+can run on MacOS but it needs users' customized installation.
 Chapter Four contains instructions for downloading and installing Metacat and the
 applications required to run the software on Linux platform.
 Chapter Five covers how to configure Metacat, both for new and upgraded
-installations. Chapter Seven details the ways in which you can customize the
-Metacat interface so users can access and submit information easily: using
-Metacat's generic web-interface (the Registry), creating your own HTML forms,
-and creating your own desktop client (like Morpho). Chapter Eight discusses how
-to work with Metacat's embedded Geoserver. Chapter Nine describes how to set up the
-Metacat's replication service, which permits Metacat servers to share data with
-each other, effectively backing up metadata and data files. Chapter Ten looks
-at the Metacat Harvester, a program that automates the retrieval of EML
-documents from one or more sites and their subsequent upload (insert or update)
-to Metacat. Chapter Eight discusses logging, Chapter Twelve contains instructions
-for creating a site map, which makes individual metadata entries available via
-Web searches. Metacat's `Java API`_ is available for developers.
+installations. Chapter Seven details how you can access and submit information easily using
+Metacatui, the default web interface that is shipped with Metacat. Chapter Eight describes
+the Metacat indexer with solr. Chapter Nine looks at Metacatui themes, and Chapter Ten discusses
+how to set up authentication as a Metacat operator. Chapter Thirteen discusses logging, and
+Chapter Fourteen contains instructions for creating a site map, which makes individual metadata
+entries available via Web searches. Additionally, Metacat's `Java API`_ is available for developers.
```

Comment:
this is a very dangerous paragraph :-) What happens when we delete a chapter and/or renumber others?

Reply:
Yes... the old description was not up to date, and I am setting that up again by being too specific. I have revised the intro statement.